### ISSUE TITLE
Files should not be installed during build

### ISSUE THREAD

**Issue by yochananmarqos** [BitPonyLLC/huekeys#23]
There should be a separate install step, files should not be installed during build. Also:

* Desktop files do not need a shebang
* The desktop file and icon should not be installed to `XDG_DATA_HOME` when not building as user
* The Exec and Icon paths reference the build directory, not where they are actually being installed:
   ```
   Exec=/build/huekeys/src/huekeys/huekeys menu
   Icon=/build/.local/share/huekeys/icon.png
   ```

**Comment by bradrf** [comment]
Yeah, I cringed a bit when I went that path. I'll fix it up to avoid those unless running as a standard user w/ a desktop setup.

**Comment by bradrf** [comment]
should be fixed in v0.5.10

**Comment by yochananmarqos** [comment]
Well, the desktop and icon can still be generated during build. I [added them manually](https://aur.archlinux.org/cgit/aur.git/commit/?h=huekeys&id=f3116e9b4c21880f3ef5a2e4047d28ff93ef84a4) for now.
